creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -8,6 +8,8 @@ const PROCESS_NAMES = Object.freeze({
   rehydrate_file: 'rehydrate_file'
 })
 
+const MAX_COUNT = 50000
```

Comment:
should we make this a config value like we do for `rehydrateMaxConcurrency`

Reply:
Agree with you on this @vicky-g but this isn't something we need to do here in a hotfix PR. This is something you can address with the scalability fixes you make to this flow 🙂